src/test/resources/features/US02_Douglas.feature

```diff
@@ -1,3 +1,4 @@
+@smoke @regression
 Feature: As a user, I should be able to access login page features.
 
```

Comment:
Good morning, Douglas! This file has conflict that needs to be resolved. I'm not sure if it's from my end or yours but the file name  "US02_Douglas.feature" doesn't match with the one on the master. I see you created this file 3/6/2023

Reply:
Maybe this file will help --> https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts

Reply:
Thanks for catching that. The issue should be resolved.